pay-api/src/pay_api/services/refund.py

```diff
@@ -178,16 +179,22 @@ def _process_cfs_refund(cls, invoice: InvoiceModel):
                         account_id=invoice.payment_account_id).save()
 
     @classmethod
-    def _publish_to_mailer(cls, invoice):
+    def _publish_to_mailer(cls, invoice: InvoiceModel):
         """Construct message and send to mailer queue."""
         receipt: ReceiptModel = ReceiptModel.find_by_invoice_id_and_receipt_number(invoice_id=invoice.id)
         invoice_ref: InvoiceReferenceModel = InvoiceReferenceModel.find_reference_by_invoice_id_and_status(
             invoice_id=invoice.id, status_code=InvoiceReferenceStatus.COMPLETED.value)
         payment_transaction: PaymentTransactionModel = PaymentTransactionModel.find_recent_completed_by_invoice_id(
             invoice_id=invoice.id)
+        message_type: str = f'bc.registry.payment.{invoice.payment_method_code.lower()}.refundRequest'
```

Comment:
can there be a payment_method_code with cc value?

Reply:
Not in normal cases. Only chance is when an online banking converts to CC after few days, but their refund shouldn't need an email as it involves CFS invoice.